code/modules/paperwork/photocopier.dm

```diff
@@ -546,47 +657,111 @@ GLOBAL_LIST_INIT(paper_blanks, init_paper_blanks())
 	default_unfasten_wrench(user, tool)
 	return ITEM_INTERACT_SUCCESS
 
-/obj/machinery/photocopier/attackby(obj/item/object, mob/user, list/modifiers, list/attack_modifiers)
-	if(istype(object, /obj/item/paper) || istype(object, /obj/item/photo) || istype(object, /obj/item/documents))
-		if(istype(object, /obj/item/paper))
-			var/obj/item/paper/paper = object
-			if(paper.is_empty())
-				insert_empty_paper(paper, user)
-				return
-		insert_copy_object(object, user)
+/obj/machinery/photocopier/item_interaction(mob/living/user, obj/item/tool, list/modifiers)
+	//if(!user.temporarilyRemoveItemFromInventory(tool))
+	//	return ITEM_INTERACT_FAILURE
```

Comment:
oops commented out code in the pr

Reply:
Removed via https://github.com/tgstation/tgstation/pull/91258/commits/23e4339b372e5cc56ec92083847289f17603319b